luigi/contrib/gcs.py

```diff
@@ -370,7 +370,12 @@ def download(self, path, chunksize=None):
 class _DeleteOnCloseFile(io.FileIO):
     def close(self):
         super(_DeleteOnCloseFile, self).close()
-        os.remove(self.name)
+        if not self.closed:
```

Comment:
Do you still need this? Who sets this property by the way? If it's inherited, won't it be always set to `True` when you call the parent's `close()` like you do on the line above?